src/app/components/admin/analysis-jobs/analysis-job.schema.json

```diff
@@ -0,0 +1,88 @@
+{
+  "fields": [
+    {
+      "key": "id",
+      "hide": true,
+      "templateOptions": {
+        "label": "Id"
+      }
+    },
```

Comment:
this schema will likely be reused for user-facing pages

Reply:
Alright, i've enabled editing for the name, annotationName, and customSettings. Not sure if the user should be able to change any of the other parameters

Reply:
you can refer to params methods in controller to check